src/main/java/core/basesyntax/model/MessageDetails.java

```diff
@@ -1,8 +1,17 @@
 package core.basesyntax.model;
 
 import java.time.LocalDateTime;
-
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "message_details")
 public class MessageDetails {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String sender;
     private LocalDateTime sentTime;
```

Comment:
I was given a comment - you need to rename such fields to 
 @Column (name = "sent_time")